docs/en/Alluxio-Storage.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: global
+title: Alluxio Storage
+nickname: Alluxio Storage
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio manages the local storage, including memory, of Alluxio workers to act as a distributed
+buffer cache. This fast data layer between user applications and the various under storages results
+in vastly improved I/O performance.
+
+The amount and type of storage for each Alluxio node to manage is determined by user configuration.
+Alluxio also supports tiered storage, which makes the system storage media aware, enabling data
+storage optimizations similar to L1/L2 cpu caches.
+
+## Configuring Alluxio Storage
+
+The easiest way to configure Alluxio storage is to use the default single-tier mode.
+
+Out-of-the-box Alluxio will provision a ramdisk on every worker and take a percentage of the
+system's total memory. This ramdisk will be used as the only storage medium allocated to each
+Alluxio worker.
+
+Alluxio storage is configured through Alluxio's configuration in `alluxio-site.properties`. See the
+[configuration docs](Configuration-Settings.html) for detailed information.
+
+A common modification to the default is to explicitly set the ramdisk size. For example, to set the
+ramdisk size to be 16GB on each worker:
+
+```
+alluxio.worker.memory.size=16GB
+```
+
+Another common change is to specify multiple storage media, such as ramdisk and SSDs. We will need
+to update `alluxio.worker.tieredstore.level0.dirs.path` to take specify each storage medium we want
+to use as a storage directory. For example, to use the ramdisk (mounted at `/mnt/ramdisk`) and 2
+SSDs (mounted at `/mnt/ssd1` and `/mnt/ssd2`):
+
+```
+alluxio.worker.tieredstore.level0.dirs.path=/mnt/ramdisk,/mnt/ssd1,/mnt/ssd2
+```
+
+After updating the storage media, we need to indicate how much storage is allocated for each storage
+directory. For example, if we wanted to use 16 GB on the ramdisk and 100 GB on each SSD:
+
+```
+alluxio.worker.tieredstore.level0.dirs.quota=16GB,100GB,100GB
```

Comment:
shall we explain how this property is related to `alluxio.worker.memory.size`?

Reply:
Updated, thanks.